extensions/podman/src/compatibility-mode.ts

```diff
@@ -0,0 +1,100 @@
+/**********************************************************************
+ * Copyright (C) 2023 Red Hat, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ***********************************************************************/
+
+import * as extensionApi from '@podman-desktop/api';
+import * as sudo from '@vscode/sudo-prompt';
```

Comment:
why do we need to use the fork of VS Code instead of https://www.npmjs.com/package/sudo-prompt ?

Reply:
VSCode fork is maintained and includes some bug fixes. The package you link has the last commit 3 years ago.

Reply:
AFAIK vscode one only contains Microsoft stuff on top of the fork
Original one is downloaded more than 10x time than the fork

Reply:
https://github.com/jorangreef/sudo-prompt/compare/master...bpasero:sudo-prompt:vscode